tests/test_reorder_offsets.c

```diff
@@ -17,11 +17,13 @@
 /* Global vars */
 int tests_run = 0;
 int nchunks;
+bool serialized;
+char* filename;
 
 
 static char* test_reorder_offsets(void) {
-  static int32_t data[CHUNKSIZE];
-  static int32_t data_dest[CHUNKSIZE];
+  int32_t data[CHUNKSIZE];
+  int32_t data_dest[CHUNKSIZE];
```

Comment:
The issues with Win comes from using large data in the stack.  Use the heap instead (via malloc) and this should be gone. Thanks to @stoni for finding this.